src/EnergyPlus/HeatBalanceSurfaceManager.cc

```diff
@@ -5175,7 +5175,9 @@ void UpdateThermalHistories(EnergyPlusData &state)
                         state.dataHeatBalFanSys->CTFTuserConstPart(SurfNum);
                 }
 
-                if (surface.ExtBoundCond > 0) continue; // Don't need to evaluate outside for partitions
+                // Don't need to evaluate outside for partitions (so continue),
+                // but do need to do remaining calculations for interzone partitions.
+                if (surface.ExtBoundCond > 0 && surface.ExtBoundCond == SurfNum) continue;
```

Comment:
interesting. This seems closely related to issue #9466, code changes in the same area anyway.   I had found that self-referencing partitions were also not reporting those outside face variables accurately, for similar reasons as here.  But the code I suggested is different.  

`if (surface.ExtBoundCond > 0 && surface.ExtBoundCond != SurfNum) `

perhaps need to get rid of the entire if -> continue, turns out the calculations can't be saved without reporting errors.

Reply:
@EnergyArchmage do you think it's worth talking about this on tomorrow's call?

Reply:
Sorry--I guess I never responded to @EnergyArchmage here.  My reason for skipping the partitions (when ExtBoundCond = SurfNum) is that the "outside" is really the centerline of those surfaces.  So, the outside flux should be zero because it is adiabatic.  That isn't the case for an interzone partition.  If the partitions also assign the outside surface, I'm not even sure what it will report because there is no outside surface solution for a partition.  That's why I felt like the correct solution was to do what I did.  What were the reporting errors that you noticed?

Reply:
The output variables Surface Outside Face Conduction Heat Transfer Rate  and Surface Average Face Conduction Heat Transfer Rate never get calculated and Surface Heat Storage Rate has erroneous values.  

I think just remove the if -> continue line altogether.  I believe I put that in way back to save calculation time but it was a mistake.

Reply:
Ok, I can remove the if -> continue line.  That's not a problem (may have to adjust the unit test--can't remember as it's been a while).  Are you saying that these output variables are not valid and should be eliminated?  Or is there some other clarification that is needed for the IO Ref?

Reply:
I think the output variables for outside face are useful. It is just that the underlying calcs need to happen for each surface.

Reply:
Ok, I'll work to eliminate that line (and the new comments that went with it) and then make sure the unit tests are still doing what they need to do.  I'll post here again once I've finished all that and it's ready for another look.

Reply:
@EnergyArchmage @Myoldmopar Ok, I've done as suggested, fixed the unit test, and clang-formatted everything.  This was just pushed up to GitHub.  Hopefully everything comes back as expected and this is good to go.